### PR TITLE
[Bug Fixed] Make query_rpc_tracker show the correct device server port and customized address

### PR DESCRIPTION
**Pull request by lmxyy**
Make query_rpc_tracker show the correct device server port and customized address. Previously, the `query_rpc_tracker` command shows a dummy port and does not support showing the customized address.
image
Now, it shows the correct port and supports the customized address.
image

Also fixed some minor typos.

### CODE REVIEW

**Review by areusch — CHANGES_REQUESTED** [review]
@lmxyy thanks for fixing this! I'm not quite sure I understand the order of events that leads to the incorrect port, though. could you explain for my understanding?

File: python/tvm/exec/rpc_server.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def main(args):
                         and ROCM compilers.",
     )
     parser.add_argument(
-        "--custom-addr", type=str, help="Custom IP Address to Report to RPC Tracker"
+        "--custom-addr", type=str, default=None, help="Custom IP Address to Report to RPC Tracker"
```
[the remark this answers is not in the document]
Comment (lmxyy) [reply python/tvm/exec/rpc_server.py:81]: Yes, by default it is `None`. But I used to explicitly define it. I will change it.

**Comment by lmxyy** [comment]
> @lmxyy thanks for fixing this! I'm not quite sure I understand the order of events that leads to the incorrect port, though. could you explain for my understanding?

Sure. Previously, when you set up a server on your device side, the device would open a new connection to the tracker. When the tracker accepts the connection, it will register the device and record where the connection comes from ([code](https://github.com/apache/tvm/blob/64a8e81eeeefee0b11669dc83ee0d9ec871270f7/python/tvm/rpc/tracker.py#L311-L312)). However, this connection is built on a dummy port on the device instead of the server port, so the address is not correct. Moreover, this address will not be updated since the device only sends the device key to the tracker ([code](https://github.com/apache/tvm/blob/64a8e81eeeefee0b11669dc83ee0d9ec871270f7/python/tvm/rpc/server.py#L219)).

**Comment by areusch** [comment]
@lmxyy ah i had missed the thing at your [first code pointer](https://github.com/apache/tvm/blob/64a8e81eeeefee0b11669dc83ee0d9ec871270f7/python/tvm/rpc/tracker.py#L170). 

could you change that line to `self._info = {}` then? there is no sense in recording the tracker connection's remote address in place of the RPC server that should be there. the "dummy" value you're referring to is the port dynamically allocated by the OS of the machine running the RPC server when the tracker connection is [established](https://github.com/apache/tvm/blob/64a8e81eeeefee0b11669dc83ee0d9ec871270f7/python/tvm/rpc/server.py#L213).

**Comment by lmxyy** [comment]
Sure, I've changed `self._info={}`.
@tqchen Could you help review this pull request?

**Comment by lmxyy** [comment]
@tqchen @areusch @merrymercy

**Review by merrymercy — APPROVED** [review]
LGTM

**Review by areusch — CHANGES_REQUESTED** [review]
ok great, this change makes sense to me. as i understand it, it primarily affects the summary retrieved with `query_rpc_tracker`. could you add a test (or modify `test_rpc_tracker_register`) to tests/python/unittest/test_runtime_rpc.py?

**Comment by lmxyy** [comment]
@areusch Do I need to make further modifications?

**Review by areusch — APPROVED** [review]
Thanks @lmxyy!

**Pull request merged**